.env.example

```diff
@@ -1,2 +1,4 @@
 TOKEN=YOUR_TOKEN
 DATABASE_URL=mongodb://localhost:27017
+SERVICE_EMAIL = "dealfinder@gmail.com"
+SERVICE_PASSWORD ="16digitgpass"
```

Comment:
Let's make it a little more generic and not add specific email and password in the example.

```
SERVICE_EMAIL=YOUR_SERVICE_EMAIL
SERVICE_EMAIL_PASSWORD=YOUR_SERVICE_EMAIL_PASSWORD
```

Reply:
done